web/pf4/src/components/KChart.tsx

```diff
@@ -101,58 +99,133 @@ class KChart extends React.Component<KChartProps, State> {
+              }
+            ];
+          },
+          onMouseOut: () => {
+            return [
+              {
+                childName: ['serie-' + idx],
+                target: 'data',
+                eventKey: 'all',
+                mutation: () => {
+                  return null;
+                }
+              }
+            ];
+          },
+          onClick: () => {
+            return [
+              {
+                childName: ['serie-' + idx],
+                target: 'data',
+                mutation: () => {
+                  if (!this.state.hiddenSeries.delete(idx)) {
+                    // Was not already hidden => add to set
+                    this.state.hiddenSeries.add(idx);
+                  }
+                  this.setState({ hiddenSeries: new Set(this.state.hiddenSeries) });
+                  return null;
+                }
+              },
+              {
+                childName: ['serie-' + idx],
+                target: 'data',
+                eventKey: 'all',
+                mutation: () => null
+              }
+            ];
+          }
+        },
+      };
+    });
+
     return (
       <div ref={this.containerRef}>
         <TextContent>
           <Text component={TextVariants.h4} style={{textAlign: 'center'}}>{this.props.chart.name}</Text>
         </TextContent>
-        <div className="area-chart-overflow">
-          <Chart
-            height={height}
-            width={this.state.width}
-            containerComponent={createContainer()}
-            themeColor={ChartThemeColor.multi}
-            scale={{x: 'time'}}
-            minDomain={minDomain}
-            maxDomain={maxDomain}>
-            <ChartGroup offset={groupOffset}>{this.props.data.series.map(seriesBuilder)}</ChartGroup>
-            <ChartAxis
-              tickCount={scaleInfo.count}
-              style={{ tickLabels: {fontSize: 12, padding: 2} }}
-            />
-            <ChartAxis
-              tickLabelComponent={<VictoryPortal><VictoryLabel/></VictoryPortal>}
-              dependentAxis={true}
-              tickFormat={getFormatter(d3Format, this.props.chart.unit)}
-              style={{ tickLabels: {fontSize: 12, padding: 2} }}
-            />
-          </Chart>
-        </div>
-        <ChartLegend
-          x={50}
-          data={legend.items}
-          height={legend.height}
-          themeColor={ChartThemeColor.multi}
+        <Chart
+          height={height}
           width={this.state.width}
-          itemsPerRow={legend.itemsPerRow}
-        />
+          padding={padding}
+          events={events}
+          containerComponent={createContainer()}
+          themeColor={ChartThemeColor.multi}
+          scale={{x: 'time'}}
+          minDomain={minDomain}
+          maxDomain={maxDomain}>
+          <ChartGroup offset={groupOffset}>
+            {this.props.data.map((serie, idx) => {
+              if (this.state.hiddenSeries.has(idx)) {
+                return undefined;
```

Comment:
Oh! I understand. I guess it's VictoryCharts who choose the line color given the `themeColor={ChartThemeColor.multi}`

This complicates things a little bit :/

Reply:
Perhaps a hack could be Opacity to 0%???